new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/primitive/HelmParams.java

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2020, Oracle Corporation and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package oracle.weblogic.kubernetes.actions.impl.primitive;
+
+import java.util.HashMap;
+
+// All common parameters needed to install a helm application
+
+public abstract class HelmParams {
```

Comment:
why an abstract class?

Reply:
Now as I removed inheritance, don't need the abstract method in this class anymore and this class is not abstract now.